drivers/firmware/Kconfig

```diff
@@ -15,4 +15,14 @@ source "drivers/firmware/nrf_ironside/Kconfig"
 source "drivers/firmware/scmi/Kconfig"
 # zephyr-keep-sorted-stop
 
+config TISCI
+	bool "TISCI Firmware driver"
+	select MBOX
+	help
+	  TISCI firmware driver is a frontend interface
+	  to TI System Controller interface firmware through
+	  secureproxy mailbox.
+
+source "drivers/firmware/ti_sci/Kconfig"
```

Comment:
move this config to that file, put source file in list, correctly sorted, add correct depends on to this symbol